CppCoreGuidelines.md

```diff
@@ -21972,23 +21972,23 @@ Never allow an error to be reported from a destructor, a resource deallocation f
 
         void test(string& s)
         {
-            Nefarious n;          // trouble brewing
+            Nefarious n;          // might throw on destruction
             string copy = s;      // copy the string
         } // destroy copy and then n
 
     Here, copying `s` could throw, and if that throws and if `n`'s destructor then also throws, the program will exit via `std::terminate` because two exceptions can't be propagated simultaneously.
 
-2. Classes with `Nefarious` members or bases are also hard to use safely, because their destructors must invoke `Nefarious`' destructor, and are similarly poisoned by its poor behavior:
+2. Classes with `Nefarious` members or bases are also hard to use safely, because their destructors must invoke `Nefarious`' destructor, and are similarly impacted by its unconventional behavior:
```

Comment:
I wouldn't think it's controversial to refer to this as "bad" behavior

Reply:
I have no idea what the problem with "poisoned" is though.

Reply:
using "bad". ("poisoned" is just an unnecessary metaphor here, that makes text a little less obvious to understand for people who don't speak English well. I can revert if people prefer to keep it).

Reply:
actually I reverted the change of metaphors, it makes the changes of this PR more difficult to understand, and does not improve the document much